met/src/tools/core/point_stat/point_stat.cc

```diff
@@ -1734,11 +1741,11 @@ void do_hira_ens(int i_vx, const PairDataPoint *pd_ptr) {
          if(gt) { delete gt; gt = 0; }
          continue;
       }
-
+      
       // Write out the ECNT line
       if(conf_info.vx_opt[i_vx].output_flag[i_ecnt] != STATOutputType_None) {
-
-         // Compute ensemble statistics
+	 
+	 //Compute ensemble statistics
```

Comment:
```suggestion
         //Compute ensemble statistics
```
It looks like a mix of tabs and spaces here